test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Csv.Tests.ps1

```diff
@@ -41,14 +41,14 @@ Describe "Import-Csv Double Quote Delimiter" -Tags "CI" {
     }
 
 
-    It "Should handle <name>" -TestCases @(
+    It "Should handle <name> and bind to LiteralPath from pipeline" -TestCases @(
         @{ name = "quote with empty value"  ; expectedHeader = "a1,H1,a3"; file = "EmptyValue.csv"      ; content = $empyValueCsv       ; delimiter = '"' }
         @{ name = "quote with value"        ; expectedHeader = "a1,a2,a3"; file = "WithValue.csv"       ; content = $withValueCsv       ; delimiter = '"' }
         @{ name = "value enclosed in quote" ; expectedHeader = "a1,a2,a3"; file = "QuotedCharacter.csv" ; content = $quotedCharacterCsv ; delimiter = ',' }
         ){
         param($expectedHeader, $file, $content, $delimiter)
         Set-Content testdrive:/$file -Value $content
-        $returnObject = Import-Csv -Path testdrive:/$file -Delimiter $delimiter
+        $returnObject = Get-ChildItem -Path testdrive:/$file | Import-Csv -Delimiter $delimiter
```

Comment:
Is there already an existing test case for the example @mklement0 brought up?

```powershell
'path' | import-csv
```

Reply:
I did not understand your question. Previously the `'path' | import-csv` example did not work.

Reply:
I'm referring to @mklement0's comment here https://github.com/PowerShell/PowerShell/pull/8277#discussion_r234360210

Reply:
The fix come from the comment. I see no reason to add an explicit test for the fix, so I simply changed the existing test.

Reply:
@iSazonov: 

The `'path' | import-csv` example _did_ work, it was just piping from `Get-ChildItem` / `Get-Item` that was broken.

Also, given that we've now added support for `[pscustomobject] @{ Path = 'path' } | Import-Csv`, I think we should test that too.

Reply:
What is new code path we cover by the test?

Reply:
I guess it comes down to whether we want to test _all_ aspects of the expected pipeline-binding behavior (I don't know what the policy is and how this is handled for other cmdlets).

If we do:

* You've already added a test for binding `Get-ChildItem` / `Get-Item` input.

* Tests currently missing:

 * `'path' | import-csv` (as before)
 * `[pscustomobject] @{ Path = 'path' } | Import-Csv` (newly added)
 * `[pscustomobject] @{ LiteralPath = 'path' } | Import-Csv` (as before)

Reply:
@iSazonov the concern is that we didn't have adequate test coverage here and the previous commit caused a regression.  If this was a big ask, I would have it as a separate PR/issue, however, it seems like a small item to add some test cases for those @mklement0 outlined above.